governing-documents/project-annual-review.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Project Annual Review
+parent: Governing Documents
+grand_parent: Hyperledger TOC
+nav_order: 5
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Project Annual Review
+
+## Overview
+In addition to the [quarterly project updates](./project-updates.md), projects will undergo an annual review by the TOC. Unlike the quarterly updates, the annual review process will include a deep dive into the project health, including contributor and maintainer diversity and adoption metrics. The annual review will also include an assessment of the current [project lifecycle state](./project-lifecycle.md). The review will result in a set of recommendations for the project to improve and/or recommendation to move a project to another stage.
```

Comment:
Q: How about converting one of the quarter into an annual review for projects?

Reply:
From reading this, I think the Annual Review is quite similar to the Quarterly Report, but:

* Includes some extra requirements — goals for the next year, review of progress on goals from the prior year, an * assessment of the threats and opportunities for the project.  So likely a quarterly report, but with forced “big picture” assessment.
* Requires the presentation of the report to the TOC with a specific invitation to the Project community.

I suggest like the Quarterly Report, there be a template defined for projects to use.

Totally agree with @arsulegai that the Annual Report replaces a Quarterly Report.

Reply:
Regarding the timing of these reviews, I wonder if it makes sense to do them all around the same time. Like, we can designate a week of the year as Hyperledger Annual Review Week. This may allow the TOC to be more focused on the assessments and to get a better picture of how the projects relate to each other; i.e., we can assess the Hyperledger portfolio as a whole.

Reply:
Updates made to reflect that the annual review will replace one of the quarterly reports. Still waiting on the poll results to determine the timing.

Reply:
It appears that most people favor doing the annual reports all within the same quarter. Updates made to reflect this.